src/PrettyPrint.cpp

```diff
@@ -127,8 +127,8 @@ std::string to_string(const vr::ETrackedDeviceProperty& value)
             return "Prop_ViveSystemButtonFixRequired_Bool";
         case vr::Prop_ParentDriver_Uint64:
             return "Prop_ParentDriver_Uint64";
-        case vr::Prop_ResourceRoot_String:
```

Comment:
I have added back in. I think I removed it due to an older version of openvr, but the new version works so I have uncommented it and pushed.